app/utils/date-util.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Returns the most recent date from an array of dates.
+ * Array may include blank values.
+*/
+export function getLatestDate(dates) {
+  if (dates) {
+    // Default JS .sort() of Date objects does not give expected results.
+    // Sorting must compare millis.
+    const sortedDates = dates.filter(date => date).sort((a, b) => b - a);
```

Comment:
Dit geeft wel een ander resultaat: voor een pubflow met een openstaande aanvraag zal de einddatum van de laatst ingevulde aanvraag worden getoond. Ik zou het als gebruiker anders verwachten. Als ge het zo bedoeld hebt, best de comment in de [publication-table-row.js](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/components/publications/overview/publications-table-row.js#L80) aanpassen.